woocommerce/woocommerce
# Fix `AddToCartForm` translation on the `Single Product` block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:







### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout trunk.
2. Switch to the Italian or Spanish language (or some language that has 100% of string translated).
3. Create a new page or post.
4. Insert the `Single Product` block and click on the `Add to Cart with Options` block.
5. Ensure that the block isn't translated.
6. Checkout this branch.
7. Repeat 3-4.
8. Ensure that the block is now translated.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Add to Cart with Options - Fix translation when used inside the Single Product block.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
